Source/ImageGallery/ImageGalleryView.swift

```diff
@@ -126,10 +125,14 @@ public class ImageGalleryView: UIView {
     collectionView.delegate = self
 
     topSeparator.frame = CGRect(x: 0, y: 0, width: totalWidth, height: Dimensions.galleryBarHeight)
-    indicator.frame = CGRect(x: (totalWidth - Dimensions.indicatorWidth) / 2, y: (topSeparator.frame.height - Dimensions.indicatorHeight) / 2,
-      width: Dimensions.indicatorWidth, height: Dimensions.indicatorHeight)
-    collectionView.frame = CGRect(x: 0, y: topSeparator.frame.height, width: totalWidth, height: collectionFrame - topSeparator.frame.height)
-    collectionSize = CGSize(width: collectionView.frame.height, height: collectionView.frame.height)
+
+    indicator.center = topSeparator.center
+
+    collectionView.frame = CGRect(x: 0, y: topSeparator.frame.height, width: totalWidth, height: frame.height - topSeparator.frame.height)
+
+    let height = max(collectionView.frame.height, 100)
+    collectionSize = CGSize(width: height, height: height)
+
```

Comment:
I like to have the let before and the properties later. It's consistent through all the Picker.